app/dashboard/volunteers/page.tsx

```
@@ -1,3 +1,28 @@
-export default function Test() {
-  return <>Hello</>;
+import { createClient } from "@/utils/supabase/server";
+import { Volunteer, columns } from "./columns";
+import { DataTable } from "./data-table";
+
+async function getData(): Promise<Volunteer[]> {
+  const supabase = createClient();
+  let { data: volunteer, error } = await supabase.from("volunteer").select("*");
+
+  if (error) {
+    console.error(error);
```

Throw the error instead of logging it.

```tsx
if (error) {
  throw error;
}
```